osbrain/agent.py

```diff
@@ -1744,8 +1751,12 @@ def _teardown(self):
     def start(self):
         super().start()
         status = self.queue.get()
-        if status == 'STARTED':
-            return
+        if status.startswith('STARTED'):
+            match = re.match(r'STARTED: (.*)', status)
+            name = ''
+            if match:
+                name = match.group(1)
+            return name
```

Comment:
No need to use regular expressions here:

```python
return status.partition(':')[-1]
```

We could also say:

```python
if not status.startswith('STARTED:'):
    raise [...]
return [...]
```

Looks nicer to me in this case.

Reply:
I've gotten too used to parsing everything with regexes. Too much use of Unix shells I guess 😜